src/index.js

```diff
@@ -19,10 +19,28 @@ class Dropzone extends React.Component {
     };
   }
 
+  componentWillMount() {
+    if (this.props.global && typeof document) {
+      document.addEventListener('dragenter', this.onDragEnter);
```

Comment:
What class methods that don't exist? You mean `document`? This code works in Chrome but there might be a more better cross browser solution?